.github/workflows/api-dotnetcore.yml

```diff
@@ -2,32 +2,42 @@ name: API (.NET 5)
 
 on:
   push:
-    branches: [master, dev, dev-alpha]
+    branches: [master, dev, test]
   pull_request:
-    branches: [master, dev, dev-alpha]
+    branches: [master, dev, test]
 
 jobs:
   build:
     runs-on: ubuntu-latest
     env:
       working-directory: ./backend
-      codeCov-token: ${{ secrets.CodeCov }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
oh, so we are dropping codecov in favour of sonar?

Reply:
No, just clean up of the config.